osquery/tables/system/posix/augeas.cpp

```diff
@@ -188,6 +217,36 @@ QueryData genAugeas(QueryContext& context) {
     }
   }
 
+  // This LIKE strategy only works because we've loaded the entire
+  // augeas system. If we ever move to loading by explicit files, this
+  // will break.
+  if (context.hasConstraint("path", LIKE)) {
+    auto paths = context.constraints["path"].getAll(LIKE);
+    std::ostringstream pattern;
+
+    for (std::string path : paths) {
+      if (path.empty()) {
+        continue;
+      }
+
+      convertWildcards(path);
+
+      pattern << "/files" << path;
+      patterns.insert(pattern.str());
```

Comment:
You can write:
```
auto pattern = "/files" + path;
patterns.insert(pattern);
```

Reply:
This code all predates this. (though my refactor copy-pasta'ed it). I can try to improve this